packages/api/.env

```diff
@@ -1 +1,8 @@
-DATABASE_URL="postgresql://root:root@localhost:5432/main-db"
\ No newline at end of file
+DATABASE_URL="postgresql://root:root@localhost:5432/main-db"
+
+GITHUB_CLIENT_ID = "e40b48326fd479c3dc42"
+GITHUB_CLIENT_SECRET = "5294c04782332725525316513f957cf657f858e7"
```

Comment:
It looks like something we don't want to have in our version control system. I would suggest leaving those two values as empty strings and writing in the docs, how to obtain them.